addons/project/views/project_project_views.xml

```diff
@@ -41,8 +41,15 @@
                     </header>
                 <sheet string="Project">
                     <div class="oe_button_box" name="button_box" groups="base.group_user">
-                        <button class="oe_stat_button" type="object" name="action_view_tasks" icon="fa-tasks">
-                            <field string="Tasks" name="open_task_count" widget="statinfo"/>
+                        <button class="oe_stat_button" type="object" name="action_view_tasks" icon="fa-check" title="Number of tasks closed compared to the total number of tasks created or updated within the past 365 days.">
+                            <div class="o_field_widget o_stat_info">
+                                <span class="o_stat_text">Tasks</span>
+                                <span class="o_stat_value">
+                                    <field name="closed_task_count"/> / 
+                                    <field name="task_count"/>
```

Comment:
I think it should be better to display that in one line to avoid having a space at the end of the line.
```suggestion
                                    <field name="closed_task_count"/> / <field name="task_count"/>
```